APCAppCore/APCAppCore/DataSubstrate/Model/APCUser+Bridge.h

```diff
@@ -55,6 +55,15 @@
                  onCompletion:(void (^)(NSError *))completionBlock;
 
 - (void) signInOnCompletion:(void (^)(NSError * error))completionBlock;
+
+/**
+ * Login a user on this device via externalId where registration was handled on a different device
+ * ExternalID property must not be nil before calling this method
+ * @param defaultEmailAddress is the email address that will be used as the basis for creating the external ID login
```

Comment:
I like your idea on having it in a central location. Originally, I was looking for a similar class like SBABridgeInfo in AppCore, does one exist?